cmd/tools/extract_actions/main.go

```diff
@@ -0,0 +1,72 @@
+package main
```

Comment:
Suggest adding a build constraint + move the file to the same package (as, for example: codegen.go)
```go
//go:build gen
```